src/components/02_atoms/Widgets/DatetimeTimestamp.js

```diff
@@ -16,7 +16,7 @@ const DatetimeTimestamp = ({
     fullWidth
     label={label}
     type="datetime-local"
-    defaultValue={value ? new Date(value * 1000).toJSON().slice(0, 19) : null}
+    defaultValue={value ? new Date(value).toJSON().slice(0, 19) : null}
```

Comment:
Wow, does this mean that the output of jsonapi changed?

Reply:
@dawehner See this, https://github.com/jsdrupal/drupal-admin-ui/pull/489#issuecomment-422696697

Reply:
Oh I see, thank you!